app/lib/search/index_simple.dart

```diff
@@ -30,6 +30,21 @@ class SimplePackageIndex implements PackageIndex {
   bool get isReady => _isReady;
 
   @override
+  Map<String, dynamic> get debugInfo {
+    final data = {
+      'packageCount': _packages.length,
+      'lastUpdated': _lastUpdated?.toIso8601String(),
+    };
+
+    if (_lastUpdated != null) {
+      data['lastUpdateDelta'] =
+          new DateTime.now().difference(_lastUpdated).toString();
```

Comment:
Since `_lastUpdated` is utc, you should also use utc for `now`.

Reply:
Doesn't affect delta, since timezones are included.

When you hit it right after an update it shows as ~seconds – not hours.